src/util/file-util.ts

```diff
@@ -0,0 +1,79 @@
+import * as fs from 'fs';
+import path from 'path';
+import { exec, execFile } from 'child_process';
+
+export async function modifyFile(
+  cmd: string,
+  oldPath: string,
+  newPath: string,
+): Promise<boolean> {
+  if (!fileExists(oldPath)) {
+    console.log(`${oldPath} does not exists!`);
+    return false;
+  }
+
+  return new Promise((resolve, reject) => {
+    exec(
+      `${cmd} ${oldPath} ${newPath}`,
+      (error: any, _stdout: any, _stderr: any) => {
+        if (error != null) {
```

Comment:
It's weird that it's complaining about stdout and stderr even though you have an underscore... maybe you can remove them entirely?